### ISSUE TITLE
Can you download the source code？

### ISSUE THREAD

**Issue by ttsite** [hawksh/Hidden-miner-builder#18]
Can you download the source code, thank you!

**Comment by hawksh** [comment]
> 
> 
> Can you download the source code, thank you!

no

**Comment by SuchyChomik** [comment]
i dont think so because if you had the source you couldd easly remove the 10% donation to the author
